### ISSUE TITLE
HTTP check should set User-Agent to something custom

### ISSUE THREAD

**Issue by jsternberg** [hashicorp/consul#1053]
Inside of our Nginx logs I'll see something like this:

```
127.0.0.1 - - [22/Jun/2015:21:42:31 +0000] "GET / HTTP/1.1" 200 668 "-" "Go 1.1 package http"
```

I'm also running a Go program on this same computer and I thought the program was acting erroneously. It turns out this is just from Consul checking the HTTP endpoint. I'd like it if Consul set a custom User-Agent so I can see "Yep, this is Consul checking the HTTP health".

Thanks.

**Comment by ryanuber** [comment]
Makes sense, tagged as an enhancement!

**Comment by highlyunavailable** [comment]
It looks like this was already resolved #951 via commit ce93fdd76bcac6d0e154f5c6be189990f74d258b which should be in 0.5.2.

**Comment by ryanuber** [comment]
Ah, yep! Closing this out.
